src/Sulu/Bundle/AdminBundle/Resources/js/components/CKEditor5/ckeditor5.css

```diff
@@ -0,0 +1,176 @@
+:root {
+    /* Custom colors */
+    --alabaster: #fafafa;
+    --alto: #ddd;
+    --black: #000;
+    --blueZodiac: #112a46;
+    --cloudBurst: #1c2945;
+    --doveGray: #6e6e6e;
+    --mirage: #151b2a;
+    --concrete: #f2f2f2;
+    --dustyGray: #999;
+    --ebonyClay: #232e44;
+    --ebonyClayDark: #232739;
+    --frenchGray: #babfc7;
+    --gold: #f8d200;
+    --gallery: #ededed;
+    --gray: #878787;
+    --java: #24b7c8;
+    --mantis: #6ac86b;
+    --mercury: #e6e6e6;
+    --mineShaft: #353535;
+    --mischka: #dee0e7;
+    --nobel: #b4b4b4;
+    --persianRed: #cf3939;
+    --pickledBluewood: #293650;
+    --roman: #d96161;
+    --scorpion: #585858;
+    --shakespeare: #52b6ca;
+    --silver: #ccc;
+    --silverChalice: #a2a2a2;
+    --stormGray: #70768b;
+    --white: #fff;
+    --wildSand: #f5f5f5;
```

Comment:
That's IMO a no-go... Can't we use the colors we have defined in `colors.scss` instead?